encrypter.cpp

```diff
@@ -0,0 +1,202 @@
+#include <iostream>
+#include <string.h>
+#include <fstream>
+#include <ios>
+#include <random>
+#include <sstream>
+
+
+//=====put_error=====//
+void put_error(const std::string& error_message)
+{
+  std::cout << error_message;
+  exit(1);
+}
+
+//=====shuffle=====//
+
+void shuffle(std::vector<char>& input)
```

Comment:
Better make type alias for Byte using "using" keyword.
I mean do not use "char" or "unsigned char" directly. Use it via type alias.
And better make it unsigned char instead of char.